src/util/DateUtils.js

```diff
@@ -80,6 +80,36 @@ export default class DateUtils {
     return '';
   }
 
+  /**
+   * Given a string, return a Date object created by parsing it.
```

Comment:
The comment mentions nothing about UTC... Does the original string represent a UTC time? If so, maybe we can call the function `utcStringToDate()`? And/or name the parameter `utcDateString`?

Can you maybe add an example of a string you might pass to it? Maybe in a @param tag?